frontend/src/api/pipelines/__tests__/errorUtils.spec.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable camelcase */
+import { handlePipelineFailures } from '~/api/pipelines/errorUtils';
+import { mockPipelineKF } from '~/__mocks__/mockPipelineKF';
+import { proxyGET } from '~/api/proxyUtils';
+import { NotReadyError } from '~/utilities/useFetchState';
+
+jest.mock('~/api/proxyUtils.ts', () => ({
+  proxyGET: jest.fn(),
+}));
+
+jest.spyOn(console, 'error').mockImplementation(jest.fn());
```

Comment:
I have added this line because it was consoling error message  in the terminal , is that fine?

Reply:
Console log is fine. Unless we want to get rid of it across all tests and do a global solution..this is likely not the first or last test to console log.